python/ray/workflow/tests/conftest.py

```diff
@@ -5,49 +5,79 @@
 
 from ray.tests.conftest import get_default_fixture_ray_kwargs
 from ray._private.test_utils import simulate_storage
+from ray.cluster_utils import Cluster
 
 # Trigger pytest hook to automatically zip test cluster logs to archive dir on failure
 from ray.tests.conftest import pytest_runtest_makereport  # noqa
 from ray.workflow.tests import utils
 
 
 @contextmanager
-def _workflow_start(storage_url, shared, **kwargs):
+def _workflow_start(storage_url, shared, use_ray_client, **kwargs):
+    assert use_ray_client in {"no_ray_client", "ray_client"}
     init_kwargs = get_default_fixture_ray_kwargs()
     init_kwargs.update(kwargs)
     init_kwargs["storage"] = storage_url
-    if ray.is_initialized():
-        ray.shutdown()
+    ray.shutdown()
+    if use_ray_client:
+        # Kill the Ray cluster.
+        subprocess.check_call(["ray", "stop"])
     # Sometimes pytest does not cleanup all global variables.
     # we have to manually reset the workflow storage. This
     # should not be an issue for normal use cases, because global variables
     # are freed after the driver exits.
-    address_info = ray.init(**init_kwargs)
+    if use_ray_client == "ray_client":
```

Comment:
why not always use Cluster and when doing init, we do things specially? (https://github.com/ray-project/ray/blob/master/python/ray/tests/conftest.py#L368)

Reply:
actually true, I believe with some tests migrated, now we can just use a cluster